colossus-tests/src/it/scala/colossus/MemcacheITSpec.scala

```diff
@@ -0,0 +1,101 @@
+package colossus
+
+import colossus.TestMemcachedServer.{GetReplies, GetReply, WriteParameters}
+import colossus.core.ServerRef
+import colossus.protocols.http.HttpCodes
+import colossus.protocols.memcache.MemcacheReply.Value
+import colossus.testkit.HttpServiceSpec
+import net.liftweb.json.Serialization.write
+
+import scala.concurrent.duration._
+/*
+Please be aware when running this test, that if you run it on a machine with a memcached server
+running on 11211, it will begin to interact with it.  Be mindful if you are running this on a server
+which has data you care about.
+This test runs by hitting the REST endpoints exposed by teh TestMemcachedServer, which just proxies directly to
+a Memcached client, which communicates with memcached
+ */
+class MemcacheITSpec extends HttpServiceSpec{
```

Comment:
So these tests seem pretty good, but I am kind of confused why we need to go through the process of starting up a http server to test a memcache client.  Since we're just testing the protocol and not any of the client connection semantics, we should be able to test with just an `AsyncServiceClient` and directly check the replies from Memcached.  Not a huge deal though.